docs/logs/faq.md

```diff
@@ -576,12 +570,11 @@ formats.
 
 **What happens if I try to send data with @timestamp older than my retention time?**
 
-You can send old logs.  However, keep in mind your logging app's
-retention settings.  For example, if your retention is set to 7 days and
+You can send old logs because our retention machanism works on received timestampl, not @timestamp.
```

Comment:
Fix these typos:
You can send old logs because our retention machanism works on received timestampl, not @timestamp.
->
You can send old logs because our data retention works by checking the received `timestamp`, not `@timestamp`.

Note:
Can you explain the difference between `timestamp` and `@timestamp`? I don't think average users know this.